packages/styles/src/components/forms.scss

```diff
@@ -51,25 +51,90 @@ select.form-control-rg:not([size]):not([multiple]) {
       border-left: inherit;
     }
 
+    // Fix Safari cutoff when change the initial position of the selector button.
+    @supports (background: -webkit-named-image(i)) {
```

Comment:
This come from: https://www.bram.us/2021/06/23/css-at-supports-rules-to-target-only-firefox-safari-chromium/#safari